modules/theme/src/theme-core/definitions/globals/product.less

```diff
@@ -62,6 +62,10 @@ body {
          Global Helper Classes
```

Comment:
We'll move this section to a new file. may be `helpers` in `globals`. not in this, in a separate PR

Reply:
+1. Lets do that.